test/topology_random_failures/cluster_events.py

```diff
@@ -589,9 +591,7 @@ async def kill_coordinator_node(manager: ManagerClient,
 
     LOGGER.info("Kill the coordinator node")
     await manager.server_stop(server_id=(await get_coordinator_host(manager=manager)).server_id)
-
-    LOGGER.info("Sleep for 2 seconds")
-    await asyncio.sleep(2)
+    await wait_new_coordinator_elected(manager=manager, expected_num_of_elections=2, deadline=time.time() + 60)
```

Comment:
Could you tell me in more detail what was the purpose of the 2-second sleep, and why you changed it? This is not explained in the commit message / PR description.

Reply:
> Also add a wait for a new coordinator election in cluster events which cause such elections.

2s sleep was here for the same reason, actually.